helixbass/tree-sitter-grep
# Sort matches

In this PR:
- fix issue where matches (specifically the captures we specified I guess) were not always in fact returned in sorted order by the underlying tree-sitter query API

To test:
If you run the new test without the corresponding changes for explicitly sorting matches then you should see the test fail (it doesn't include all expected results because we violated our own invariant that our matches are stored in source-text order)
